### PR TITLE
Add PHP 8 compatibility

### PR DESCRIPTION
**Pull request by fxbt**
Tested on PHP 7.4.23 and PHP 8.0.10

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
     "require": {
         "php": ">=7.0",
         "ext-json": "*",
-        "elasticsearch/elasticsearch": "^5.1.0||^6.0.0"
+        "elasticsearch/elasticsearch": "^5.1.0||^6.0.0||^7.11"
```
Comment (Oliboy50) [inline composer.json:15]:
```suggestion
        "elasticsearch/elasticsearch": "^5.1||^6.0||^7.0"
```

except if versions between `7.0.0` and `7.11` are really broken?

**Review by Oliboy50 — APPROVED** [review]
🤞

File: composer.json (hunk 1 above)
Comment (fxbt) [reply composer.json]: PHP8 is only supported from `^7.11` but you're right, there's no reason to skip the `7.0`

I updated the branch

**Comment by fxbt** [comment]
I don't want to be rude but any update on this pull request ?

**Review by jdecool — APPROVED** [review]

**Comment by Oliboy50** [comment]
thanks @fxbt 👌

**Pull request merged**